classes/ObjectModel.php

```diff
@@ -2196,4 +2196,18 @@ public function getHtmlFields()
 
         return self::$htmlFields[$this->def['table']];
     }
+
+    /**
+     * @param array $fields
+     */
+    public function addFieldsToUpdate(array $fields): void
+    {
+        if (null === $this->update_fields) {
+            $this->update_fields = [];
+        }
+
```

Comment:
```suggestion
```

Reply:
update_fields is null|array type, so we need to initiate the array before trying to append any item to it.